plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyAgentClientImpl.java

```diff
@@ -138,7 +140,9 @@ public CompletableFuture<Void> secrets_delete(Secrets_DeleteParams params) {
   }
 
   @Override
-  public void debug_message(DebugMessage params) {}
+  public void debug_message(DebugMessage params) {
+    log.info(String.format("%s: %s", params.channel, params.message));
```

Comment:
This does lead to every debug message showing up in the Cody Log twice (once as info and once as a sent message)

![image](https://github.com/user-attachments/assets/839a8aee-1b15-4514-a821-2ee4f95941db)

Reply:
Can you take a look at this? [CODY-3559: Remove webview protocol logs from "Cody Logs"](https://linear.app/sourcegraph/issue/CODY-3559/remove-webview-protocol-logs-from-cody-logs)

Reply:
Sure!